docs/filesystems.txt

```diff
@@ -0,0 +1,144 @@
+.. _laravel-queues:
+
+======
+Queues
+======
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, gridfs, code example
+
+Overview
+--------
+
+Using MongoDB to store files can be done using the
+`GridFS Adapter for Flysystem <https://flysystem.thephpleague.com/docs/adapter/gridfs/>`__.
+GridFS lets you store files of unlimited size in the same database as your data.
+This ensures the integrity of transactions and backups.
```

Comment:
Driver implementations of GridFS do not support transactions ([DRIVERS-2062](https://jira.mongodb.org/browse/DRIVERS-2062)).